soco/core.py

```diff
@@ -432,20 +432,43 @@ def ramp_to_volume(self, volume, ramp_type='SLEEP_TIMER_RAMP_TYPE'):
         return int(response['RampTime'])
 
     @only_on_master
-    def play_from_queue(self, index, start=True):
-        """ Play a track from the queue by index. The index number is
-        required as an argument, where the first index is 0.
+    def jump_to_track(self, index):
+        """ Jump to another track in the queue.
+        This function can only be used if the player is already set to use the
+        queue. If not use `play_from_queue`. This is faster than
+        `play_from_queue` if already using the queue, as it does not reinstate
+        the queue.
 
-        index: the index of the track to play; first item in the queue is 0
-        start: If the item that has been set should start playing
+        Note:
+            If the queue is already playing it will continue to play using the
+            new track after jump_to_track. If paused it will remain paused.
 
-        Returns:
-        True if the Sonos speaker successfully started playing the track.
-        False if the track did not start (this may be because it was not
-        requested to start because "start=False")
+        Arguments:
+            index (int): The index of the track to play; first item in the
+                queue is 0.
 
-        Raises SoCoException (or a subclass) upon errors.
+        Raises:
+            SoCoUPnPException: UPnP Error 701 if the queue isn't currently
+                playing.
+                UPnP Error 711 if the index is invalid.
```

Comment:
While I think it is a good idea to separate the error numbers like this, have you checked how it looks when the docs compile. Sphinx sometimes does weird stuff.